tests/support_server/wait_support_server_network.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2016-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Wait for support server network
+# Maintainer: Petr Cervinka <pcervinka@suse.com>
+
+use base 'opensusebasetest';
+use strict;
+use testapi;
+use serial_terminal 'select_virtio_console';
+
+sub run {
+    select_virtio_console();
+
+    # Check if network is available
+    for (1 .. 5) {
```

Comment:
I don't know if a `for` or a `while` is better, but `for` is fine for me.

Reply:
I just used, what we have in ganglia_client already :)

Reply:
Ok for me ;-)